icu4c/source/tools/toolutil/pkg_genc.h

```diff
@@ -48,7 +48,7 @@
  * the data to generate the final data library. This can
  * increase the performance of the pkdata tool.
  */
-#if U_PLATFORM == U_PF_OS400
+#if U_PLATFORM == U_PF_OS400 || U_PLATFORM_HAS_WIN32_API
```

Comment:
I am not sure about this change; where is USE_SINGLE_CCODE_FILE used?
I am nervous about changing behavior for Windows users who might rely on the current behavior.
@rp9-next @srl295

Reply:
Oh, I thought this was used for the MSVC packaging. I skipped it during my Meson tests because it needs an option file, which is hard to generate for MinGW. (This is why I went the genccode way, and found the other bugs I reported.)

If it's not needed, or will introduce undesired changes, let me know and I'll remove it.